src/sql/ob_sql.cpp

```diff
@@ -1530,7 +1530,7 @@ int ObSql::handle_pl_execute(const ObString &sql,
   if (OB_FAIL(ret)) {
   } else if (OB_ISNULL(pctx)) {
     ret = OB_INVALID_ARGUMENT;
-    LOG_WARN("invalid arguement", K(ret));
+    LOG_WARN("invalid argument", K(ret));
```

Comment:
error: src/sql/ob_sql.cpp: patch does not apply
please remove the modification of this file.

Reply:
ok